Remotion/Data/DomainObjects/Infrastructure/ObjectIDConverter.cs

```diff
@@ -0,0 +1,83 @@
+// SPDX-FileCopyrightText: (c) RUBICON IT GmbH, www.rubicon.eu
+// SPDX-License-Identifier: LGPL-2.1-or-later
+using System;
+using System.ComponentModel;
+using System.Globalization;
+using Remotion.Utilities;
+
+namespace Remotion.Data.DomainObjects.Infrastructure;
+
+/// <summary> Specialization of <see cref="TypeConverter"/> for conversions from and to <see cref="ObjectID"/>. </summary>
+public class ObjectIDConverter : TypeConverter
+{
+  /// <summary> Test: Is <paramref name="sourceType"/> a <see cref="String"/>? Only <see cref="String"/>s can be converted to <see cref="ObjectID"/>s.</summary>
+  /// <param name="context"> An <see cref="ITypeDescriptorContext"/> that provides a format context. </param>
+  /// <param name="sourceType"> The <see cref="Type"/> of the value to be converted into an <see cref="ObjectID"/> type. </param>
+  /// <returns> <see langword="true"/> if the conversion is supported. </returns>
```

Comment:
As these are implementations we don't need to reintroduce the XML docs. We can add `/// <inheritdoc />` if you want but I think that is also the default so...